lab_1_classify_by_unigrams/main.py

```diff
@@ -57,51 +129,57 @@ def detect_language(
         profile_2: dict[str, str | dict[str, float]],
 ) -> str | None:
     """
-    Detects the language of an unknown profile
-    :param unknown_profile: a dictionary of a profile to determine the language of
-    :param profile_1: a dictionary of a known profile
-    :param profile_2: a dictionary of a known profile
-    :return: a language
-    """
+        Detects the language of an unknown profile
+        :param unknown_profile: a dictionary of a profile to determine the language of
+        :param profile_1: a dictionary of a known profile
+        :param profile_2: a dictionary of a known profile
+        :return: a language
+        """
+    if type(unknown_profile) != dict or type(profile_1) != dict or type(profile_2) != dict:
+        return None
+    if compare_profiles(unknown_profile, profile_1) < compare_profiles(unknown_profile, profile_2):
+        return profile_1['name']
+    else:
```

Comment:
`else` here is redundant. do you see why?

Reply:
I changed it and now it's without else